README.md

```diff
@@ -103,4 +103,14 @@ Now visit [`http://localhost:5100/`](http://localhost:5100/) in your web browser
 docker-compose up --build 
 ```
 
-Now visit [`http://localhost:8080/`](http://localhost:8080/) in your web browser to view the app.
\ No newline at end of file
+Now visit [`http://localhost:8080/`](http://localhost:8080/) in your web browser to view the app.
+
+## Travis CI
+#### Set up Travis CI for your repository
```

Comment:
Nice docs! We should also add some documentation above for how to run the tests inside docker now that that's an option (particularly since some of the changes will stop them from working on many machines locally!)